Ex2/ex2.js

```diff
@@ -0,0 +1,26 @@
+//Scrieți o funcție pentru a genera un număr aleatoriu mai mare sau egal cu o variabilă start și strict mai mic decât o variabilă end . 
```

Comment:
se recomanda dupa comentarii de lasat un spatiu liber
```suggestion
// Scrieți o funcție pentru a genera un număr aleatoriu mai mare sau egal cu o variabilă start și strict mai mic decât o variabilă end . 
```